mitype/app.py

```diff
@@ -371,7 +371,7 @@ def update_state(self, win):
         )
         if index == len(self.text):
             # Highlight mistyped characters
-            for i in set(self.mistyped):
+            for i in set(self.mistyped_keys):
```

Comment:
Can we remove the set and use the list as is? Some duplicates will be reprinted but that wouldn't affect the output.

Reply:
Sure thing, I agree that duplicates will probably appear rarely and it's not an issue to print the occasional duplicate more than once.